tasks/repack/repack.go

```diff
@@ -19,22 +22,59 @@ func configure() {
 	core.ConfigureApp()
 }
 
-func Repack(table string) error {
-	// TODO
+// GetCmd returns command that calls gp_repack with table. Table must be a partitioned table.
+// Args are appended to the necessary pgRepackArgs. Stdout and stderr of the subprocess are redirected to host.
+func getCmd(table string, args ...string) *exec.Cmd {
+	fullArgs := pgRepackArgs
+	fullArgs = append(fullArgs, "-I", table)
+	fullArgs = append(fullArgs, args...)
+	cmd := exec.Command("pg_repack", fullArgs...)
+	cmd.Env = append(os.Environ(), fmt.Sprintf("PGPASSWORD=%s", utils.CoreCfg.DBPassword))
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd
+}
+
+// Repack runs pg_repack reindex with table. If columns are provided, cluster by these columns is executed as well.
+// Table must be a partitioned table.
+func Repack(table string, columns string) error {
+	reindexCmd := getCmd(table, "-x")
+	err := reindexCmd.Run()
+	if err != nil {
+		return err
+	}
+
+	if columns == "" || len(columns) == 0 {
```

Comment:
if len(columns) == 0 should be enough since len("") = 0

Reply:
Yeah, IDK how this got in there 😅